CHANGELOG.md

```diff
@@ -4,6 +4,20 @@ This file contains notable changes (e.g. breaking changes, major changes, etc.)
 
 This file was introduced starting Kani 0.23.0, so it only contains changes from version 0.23.0 onwards.
 
+## [0.30.0]
+
+## What's Changed
```

Comment:
Can you please add this change as well? https://github.com/model-checking/kani/commit/faa4086e6d7be8fa2a363d6c7d7c0f46607a6429

Reply:
done